keyboards/handwired/tractyl_manuform/5x6_right/5x6_right.h

```diff
@@ -26,6 +26,8 @@
 #    include "teensy2pp.h"
 #elif defined(KEYBOARD_handwired_tractyl_manuform_5x6_right_elite_c)
 #    include "elite_c.h"
+#elif defined(KEYBOARD_handwired_tractyl_manuform_5x6_right_arduinomicro)
+#    include "arduinomicro.h"
```

Comment:
arduino micro, or pro micro?

Reply:
In fact naming is hard here. The keyboard uses an Arduino micro for the right side with the trackball and an Arduino pro micro for the left. But both sides are flash able with the same hex file. 
I chose the pro micro for the left because of cost reason.